src/components/SearchQueryFetchConnector/SolrQueryFetchConnector.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import SearchForm from '../SearchForm/SearchForm';
```

Comment:
This should use an "absolute" import, e.g.

`import SearchForm from 'SearchForm';`

https://medium.com/beqode/absolute-vs-relative-import-paths-nodejs-1e4efa65a7bb